src/components/areaMenu/AposAreaMenu.vue

```diff
@@ -12,21 +12,25 @@
             <dt>
               <button
                 :for="item.label" class="apos-area-menu__group-label"
-                v-if="item.items" tabindex="0" :id="menuId + '-trigger-' + index"
+                v-if="item.items" tabindex="0"
+                :id="menuId + '-trigger-' + index"
```

Comment:
Should we be using template literals? Besides this line there are also instances on `17`, `40`, `41`, `48`, `54`, `55`, `65`, `66`.

Reply:
I have no problem with that. That wasn't a linter error or a change in this PR, so I'd prefer to manage that in a separate one.